utils/sentry_client.py

```diff
@@ -0,0 +1,155 @@
+import requests
+import json
+
+class SentryClient:
+  ORGANIZATION = "sentry"
+
+  def __init__(self, host, token):
+    self.host = host
+    self.auth_token = token
+
+  def do_sentry_api_call(self, method, path, slugs, payload=None):
+    success_codes = {"get": 200, "post": 201, "delete": 204, "put": 200}
+    url = "%s/api/0/%s/" % (self.host, path)
```

Comment:
we tend to prefer the f-strings, or at `format` over the `%` formatting, which is very py2. I realize this is not in the pystyle guide by @apahim. Do you agree?

Reply:
That should go into the style guide for sure :)

Reply:
No problems using f-strings.  Style guide should probably call out using upper or lower case f as well as single or double quoted strings. :)